csm/core/controllers/users.py

```diff
@@ -106,12 +79,15 @@ class CsmGetUsersSchema(Schema):
                                                       'username',
                                                       'user_type',
                                                       'email',
+                                                      'role',
                                                       'created_time',
                                                       'updated_time']),
                              default="user_id",
                              missing="user_id")
     sort_dir = fields.Str(validate=validate.OneOf(['desc', 'asc']),
                           missing='asc', default='asc')
+    username = fields.Str(default=None, missing=None, validate=validate.Length(min=1, max=56))
```

Comment:
Any chance that we can replace the magic number on length validation by an (existing!) business logic that gives us the max username length?

Reply:
Done.

Reply:
There is a `const.CSM_USER_LENGTH_MIN_LENGTH` as well, maybe we should user it for the minimum length.

Reply:
The thing is, that's actually not a full name, but a pattern to match, this is why the min is 1.